src/AddTodoForm.jsx

```diff
@@ -1,18 +1,23 @@
+import {useState} from "react";
+
 const AddTodoForm = ({onAddTodo}) => {
+    const [todoTitle, setTodoTitle] = useState("")
 
+    const handleTitleChange = (e) => {
+        const value = e.target.value;
+        setTodoTitle(value);
+    }
     const handleAddTodo = (e) => {
         e.preventDefault();
-        const todoTitle = e.target.elements.title.value;
-        onAddTodo(todoTitle);
-        e.target.elements.title.value = ''
-
+        onAddTodo({title: todoTitle, id: Date.now()});
+        setTodoTitle(" ")
     }
 
     return (
         <>
             <form onSubmit={handleAddTodo} action="">
```

Comment:
Not related to this MR, but we probably don't need to assign the action property here at all: https://www.w3schools.com/tags/att_form_action.asp

This property is used for submitting form data to another page, but since we call `e.preventDefault()` inside of the `handleAddTodo` function, our page will never reload and form data isn't sent anywhere!